src/Shaders/sprites.vertex.fx

```diff
@@ -40,9 +39,12 @@ void main(void) {
 	vColor = color;
 	
 	// Texture
-	vec2 uvOffset = vec2(abs(offset.x - cellInfo.x), 1.0 - abs(offset.y - cellInfo.y));
+	vec2 uvOffset = vec2(abs(offset.x - inverts.x), abs(1.0 - offset.y - inverts.y));
+	vec2 uvPlace = cellInfo.xy;
+	vec2 uvSize = cellInfo.zw;
 
-	vUV = (uvOffset + cellInfo.zw) * uvScale;
+	vUV.x = uvPlace.x + uvSize.x * uvOffset.x;
+	vUV.y = 1.0 - (uvPlace.y + uvSize.y * uvOffset.y);

```

Comment:
I think this is the only problem:
```
vUV.y = (uvPlace.y + uvSize.y * uvOffset.y);
```
Because the sprite texture is not loaded with invertY ;)

Reply:
See I knew it was something stupid :)

Reply:
Not at all!